frontend/src/App.js

```diff
@@ -3,7 +3,7 @@ import { BrowserRouter } from "react-router-dom";
 import { BrowserRouter as Router, Switch } from "react-router-dom";
 import { Client as Styletron } from "styletron-engine-atomic";
 import { Provider as StyletronProvider } from "styletron-react";
-import { PublicRoute, ProtectedRoute } from "./customRoutes/CustomRoutes";
+import { PublicRoute, ProtectedRoute } from "./CustomRoutes/CustomRoutes.js";
```

Comment:
.js extendsion is not needed in imports. create and index.js file in CustomRoutes folder. and export publicRoute and protectedRoute from there